contracts/src/v0.8/l2ep/dev/scroll/ScrollCrossDomainForwarder.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {TypeAndVersionInterface} from "../../../interfaces/TypeAndVersionInterface.sol";
+// solhint-disable-next-line no-unused-import
+import {ForwarderInterface} from "../interfaces/ForwarderInterface.sol";
+
+/* ./dev dependencies - to be moved from ./dev after audit */
+import {CrossDomainForwarder} from "../CrossDomainForwarder.sol";
+import {CrossDomainOwnable} from "../CrossDomainOwnable.sol";
+
+import {IScrollMessenger} from "@scroll-tech/contracts/libraries/IScrollMessenger.sol";
+import {Address} from "../../../vendor/openzeppelin-solidity/v4.7.3/contracts/utils/Address.sol";
+
+/**
+ * @title ScrollCrossDomainForwarder - L1 xDomain account representation
```

Comment:
Please use the `///` comment format over `/* */`

Reply:
Fixed!

Reply:
You can remove the empty lines

```
  ///
  /// @dev forwarded only if L2 Messenger calls with `xDomainMessageSender` being the L1 owner address
  /// @inheritdoc ForwarderInterface
  ///
```

```
  /// @dev forwarded only if L2 Messenger calls with `xDomainMessageSender` being the L1 owner address
  /// @inheritdoc ForwarderInterface
```

Reply:
Sounds good - removed these as well from the scroll contracts